modules/reference/pages/command/springbootUtility-thin.adoc

```diff
@@ -35,6 +35,8 @@ springBootUtility thin [options]
 
 == Options
 
+The following options are available for the `springBootUtility thin` command:
+
```

Comment:
add a title for the options table as well:

 .Options for the springBootUtility thin command